user-guide/Cloud_Platform/Catalog/Best_Practices_When_Documenting_Catalog_Items.md

```diff
@@ -0,0 +1,161 @@
+
+## Overview section
+
+**Purpose:** Summarize what makes the item valuable and why users should deploy it. Address the problems it solves and its primary benefits.
+
+**Format:** One introductory paragraph with up to 3 supporting paragraphs, of 3 to 4 sentences each.
+
+**Do's:**
+
+- Begin with a concise **introductory paragraph** focused on the item’s value.
+- Add **up to 3 additional paragraphs** elaborating on features or key use cases.
+- Keep the tone **professional**, focused on **motivating** user adoption.
+- Highlight important points with **bold text**.
+- **Organize content** from broad benefits to specific features and practical applications.
+- Use **accessible language** for both technical and non-technical readers.
+- Make use of visual **[alerts](xref:CTB_Markdown_Syntax#alerts)** where applicable.
+
+**Don'ts:**
+
+- Avoid **excessive technical details**; link to reference documentation instead.
+- Refrain from jargon or **overly complex language** that could obscure the message.
+- Avoid **excessive** amounts of **bold text and/or alerts**, as these then lose their purpose of drawing the reader's attention.
+
+## Key Features section
+
+**Purpose:** **Product-centric**. Outline the main features that distinguish the item, focusing on functionality and unique benefits.
+
+**Format:** Maximum of 5 features. Focus on those features that are most relevant for the user: quality over quantity.
+
+**Do's:**
+
+- Use direct, **benefit-oriented** language.
+- Ensure that each feature relates to **specific value** for the user.
+- Use **action verbs** (such as "Monitor", "Track", etc.).
+- Prioritize features that **differentiate** the item.
+
+**Don'ts:**
+
+- Avoid **excessive detail** or vague descriptors like "high-performance" without specific context.
+- Avoid adding key features that **do not specifically relate to the item**, but that are general to DataMiner (for example, the benefits of having alarms for your equipment, which is already a key feature of DataMiner itself).
+
+## Use Cases section
+
+**Purpose:** **User-centric**. Demonstrate practical, real-world scenarios where the solution provides value.
+
+**Format:** Provide a link to a [use case on DataMiner Dojo](https://community.dataminer.services/use-cases/) or include 2 or 3 bullet points summarizing use cases.
+
+**Do's:**
+
+- Connect examples to **common challenges** that users face (e.g. remote connectivity, high data usage).
+- Use **specific, relatable examples**, such as "Monitor remote satellite terminals in real-time".
+
+**Don'ts:**
+
+- Avoid **hypothetical scenarios**. Make sure the examples are relevant to the typical user base, showing the value of the item.
+- Do not **duplicate** points covered in the **Key Features** section.
+
+## Prerequisites section
+
+**Purpose:** List essential technical requirements needed for deployment.
+
+**Format:** 1-3 concise bullet points, limited to information that is strictly necessary.
```

Comment:
Should we include the 1-3 bullet points limitation? If there are more requirements (e.g. soft-launch option, DataMiner version, DxM version, and license), it's best if each requirement gets a bullet point of its own, whereas this format requirement makes it seem as if this should be condensed into 3 bullet points at most.

Reply:
I wanted to indicate that we should not see very small prerequisites that are not really necessary. For those things, we should share more details in the Docs. I removed the explicit number.